resources/api/reservation.py

```diff
@@ -502,9 +512,12 @@ class ReservationViewSet(munigeo_api.GeoModelAPIView, viewsets.ModelViewSet, Res
         .prefetch_related('catering_orders').prefetch_related('resource__groups').order_by('begin', 'resource__unit__name', 'resource__name')
 
     serializer_class = ReservationSerializer
-    filter_backends = (DjangoFilterBackend, filters.OrderingFilter, UserFilterBackend, ReservationFilterBackend,
-                       NeedManualConfirmationFilterBackend, StateFilterBackend, CanApproveFilterBackend)
+    filter_backends = (DjangoFilterBackend, filters.OrderingFilter, filters.SearchFilter, UserFilterBackend,
+                       ReservationFilterBackend, NeedManualConfirmationFilterBackend, StateFilterBackend,
+                       CanApproveFilterBackend)
     filterset_class = ReservationFilterSet
+    search_fields = ('user__first_name', 'user__last_name', 'user__email',
```

Comment:
What's the use case for the generic search filter? Currently the search filters for Reservation are in ReservationFilterSet (https://github.com/City-of-Helsinki/respa/blob/master/resources/api/reservation.py#L400) and partially overlapping what this new `search` filter is doing.

Can the use case be satisfied through ReservationFilterSet?

Reply:
ah, okay, I think I can extend `ReservationFilterSet` then, thanks! 🙂

Reply:
This will be introduced in a separate PR.